content/en/docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -201,6 +201,39 @@ them according to its `updateStrategy`.
 
 You can [perform a rolling update](/docs/tasks/manage-daemon/update-daemon-set/) on a DaemonSet.
 
+While working with the rolling updates of the DaemonSets, you will come across two important parameters: `maxSurge` and `maxUnavailable`.
+
+### MaxSurge 
+
+This parameter is responsible for determining the maximum number of pods that can be created above the desired number of pods mentioned in the DaemonSet.
```

Comment:
Is this true?
```suggestion
If you set this parameter to a number above 0, you allow the DaemonSet controller
to create temporary duplicate Pods during a rollout.
Because a DaemonSet aims to have one healthy Pod per node, this field also implies
the number of _nodes_ that can have a duplicate Pod on them at any one moment
during the rollout.
```